components/automate-ui/src/app/page-components/search-bar/search-bar.component.ts

```diff
@@ -310,6 +310,8 @@ export class SearchBarComponent implements OnChanges {
       this.suggestions = List<SuggestionItem>(type.providedValues);
       this.suggestionsVisible = true;
       this.isLoadingSuggestions = false;
+    } else {
+      this.suggestValues.emit({ detail: { text: '', type: type.type } });
```

Comment:
With that `FilterType` I mentioned above, you could then use that to enforce strong typing on this line. Right now, if you make a typo--e.g. change `text` to `txet`--the compiler will not complain. But with strong typing it can.